### PR TITLE
Add option for applying format changes

### PR DESCRIPTION
**Pull request by XLegland**
depends on #60

### CODE REVIEW

**Comment by gdevillele** [comment]
looks good :)

File: ci/README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-# Dagger
+# Go commands
+In order to use the following go commands Docker must be running. You can build them with by going into the `ci` subdirectories and running `go build ./...`. For running them you stay in the same directory and use `go run ` followed by the command name and its potential arguments.
```
Comment (gdevillele) [inline ci/README.md:2]: > Docker must be running

And Go must be installed on the machine. (later, we'll provide commands for full docker-contained execution)

**Review by gdevillele — APPROVED** [review]
Thanks, it looks good ! 
I only suggested changes in the README. What do you think?

File: ci/README.md (hunk 1 above)
Comment (XLegland) [reply ci/README.md:2]: I'll change it thanks

**Review by gdevillele — APPROVED** [review]
LGTM 👍 Thanks.

**Pull request merged**